.github/workflows/build-python-pdoc.yml

```diff
@@ -0,0 +1,51 @@
+name: DiskANN Build PDoc Documentation
+on: [workflow_call]
+jobs:
+  build-reference-documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
I am not sure why we need to setup python but the file I used as a template did this.

Reply:
setup python just means "make sure the right version of python is on the path so you can say `python` and get the one you expect".  I'd use 3.11